agent/src/main/java/io/amaze/bench/runtime/actor/ActorBootstrap.java

```diff
@@ -53,15 +53,15 @@
     private volatile RuntimeActor actor;
 
     ActorBootstrap(@NotNull final AgentClusterClientFactory clientFactory) throws IOException, ValidationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from constructor's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'io.amaze.bench.cluster.actor.ValidationException', as it cannot be thrown from constructor's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)